admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/dto/DynamicFieldDto.java

```diff
@@ -47,5 +49,7 @@ public class DynamicFieldDto {
 	@NotBlank
 	private String description;
 
+	@DynamicCodeValidator
+	@DynamicValueValidator
```

Comment:
Why are we not validating code and value in single validator? is it required to have 2 different validators?

Reply:
removed two different validators and used the single validator for code and dynamic fields